R/data_rename.R

```diff
@@ -10,23 +10,23 @@
 #'   pipe-workflow.
 #'
 #' @param data A data frame, or an object that can be coerced to a data frame.
-#' @param pattern Character vector.
+#' @param select Character vector.
 #'   - For `data_addprefix()` or `data_addsuffix()`, a character string, which
 #'     will be added as prefix or suffix to the column names.
```

Comment:
For `data_addprefix()`, the argument is still named `pattern` (which is intended). We could just `@inheritParams extract_column_names` for the `select` docs, and then (like in `data_select()`) add a note like:

```
#' @details
#'
#' Specifically for `data_select()`, `select` can also be a named character
#' vector. In this case, the names are used to rename the columns in the
#' output data frame. See 'Examples'.
```

(replace `data_select()` with `data_rename()`)